packages/api/next.config.js

```diff
@@ -12,6 +12,14 @@ const nextConfig = {
     // See: https://github.com/gregberge/svgr
     svgr: false,
   },
+  experimental: {
+    outputFileTracingExcludes: {
+      '*': [
+        'node_modules/@swc/core-linux-x64-gnu',
+        'node_modules/@swc/core-linux-x64-musl',
+      ],
+    },
+  },
```

Comment:
This prevents vercel from including these files in the build. These are the two biggest ones that push us over the limit. I'm not sure why they are included, they are build tools